app/assets/stylesheets/account_activations.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
```

Comment:
File looks empty besides comments... Can't remember if the book instructs the creation of the empty file and leaves it here in this chapter?

Reply:
It seems to have been generated by the account mailer scaffolding. I can remove it but not too sure if it will be used later on ?